woocommerce/woocommerce-ios
# Products Onboarding: Adds Main Analytics Hub View

part of #8148 

# Why

This PR adds an empty `AnalyticsHub` view. It consists of only placeholders so we can unblock easier parallel work.

# Demo

https://user-images.githubusercontent.com/562080/202862923-17839da1-6c1a-4826-b051-49a96b5503ec.mov


# Testing Steps

- Launch the app
- Tap the "See More" button from the main dashboard
- See the template hub view

---
